packages/cli/lib/projectLogsManager.js

```diff
@@ -0,0 +1,132 @@
+const { getProjectConfig, ensureProjectExists } = require('./projects');
+const {
+  fetchProjectComponentsMetadata,
+} = require('../../../../hubspot-local-dev-lib/dist/api/projects');
```

Comment:
Leaving a note here to change this before merging.

Reply:
Whoops, great catch.  My IDE auto import went wild with this one.

Reply:
https://github.com/HubSpot/hubspot-cli/pull/1131/commits/f88732c070b9f991f8fbdd5ee8094ec353328126